### ISSUE TITLE
A question about the MochaWeb check in examples..

### ISSUE THREAD

**Issue by brugnara** [meteor-velocity/velocity-examples#43]
Why in the examples is not used the more simple syntax:

```js
MochaWeb && MochaWeb.testOnly(function(){
```

instead of:

```js
if (!(typeof MochaWeb === 'undefined')){
  MochaWeb.testOnly(function(){
```

Are there some technical issues I did not considered? Thanks.

**Comment by Sanjo** [comment]
> Are there some technical issues I did not considered? Thanks.

No. With the latest version of mike:mocha you can even just use `MochaWeb.testOnly(function(){` without problems I think.
